lib/emerald/nodes/TagStatement.rb

```diff
@@ -6,17 +6,16 @@
 
 # A tag
 class TagStatement < Node
-  def to_html(_context)
-    "<#{elements[0].text_value}>#{elements[2].text_value}</#{elements[0].text_value}>"
-  end
-
-  def to_html_tag(context)
-    if !elements[2].empty?
-      "<#{elements[0].text_value} " +
-        elements[2].to_html(context) +
-        '>'
+  def to_html(context)
+    if !elements[3].empty?
```

Comment:
Oh shit does this work for checking if an optional thing is present? I've added some `.text_value.length.nonzero?`s lying around, I should get rid of those